tests/RunTests.cmd

```diff
@@ -172,18 +172,18 @@ IF NOT DEFINED GACUTILEXE64 IF EXIST "%WINSDKNETFXTOOLS%x64\gacutil.exe" set GAC
 set FSC=%FSCBINPATH%\fsc.exe
 set PATH=%FSCBINPATH%;%PATH%
 
-set FSCVPREVBINPATH=%X86_PROGRAMFILES%\Microsoft SDKs\F#\3.1\Framework\v4.0
+set FSCVPREVBINPATH=%X86_PROGRAMFILES%\Microsoft SDKs\F#\4.0\Framework\v4.0
```

Comment:
I don't know think that I agree that end to end testing of compiler round trip compatibility is the right way to go.  We should verify that the compiler emits correct metadata and that for established items they don't change.  But finding and running old versions of the compiler is just not going to work when we are cross platform.  Unless we verify it against the Xamarin version of F# 3.1 on mono.  But even then, I wouldn't know what to do if the mono version said something was wrong, and the windows one said everything was fine.

I think we should add a work item, to improve the testing of metadata changes.